python-sdk/src/astro/files/locations/http.py

```diff
@@ -11,12 +11,18 @@ class HTTPLocation(BaseFileLocation):
     """Handler http location operations"""
 
     location_type = FileLocation.HTTP
+    # Google Drive uses HTTP location
+    supported_conn_type = {"google_cloud_platform"}
```

Comment:
Do we even need to add this here if we are not validating the connection type for HTTP?

Reply:
Removed it it was part of some local change. [Done](https://github.com/astronomer/astro-sdk/pull/1478/commits/50bd36d6d59208d166c01b6c00eec416cebc6e15)